ai_ml_profiling/qsub_profile_polaris_sequence_parallelism.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -l
+#PBS -l select=2:system=polaris
+#PBS -l place=scatter
+#PBS -l walltime=0:10:00
+#PBS -q debug
+#PBS -l filesystems=home:eagle
+#PBS -A datascience
+#PBS -N NSYS_1x2
+#PBS -k doe
+#PBS -o /home/hossainm/hpc_workshop_october_2024/outdir
+#PBS -e /home/hossainm/hpc_workshop_october_2024/errordir
+#PBS -j oe
+
+
+# What's the benchmark work directory?
+WORK_DIR=/home/hossainm/hpc_workshop_october_2024
```

Comment:
I would suggest removing your username the RHS of the environment variables, and make it clearer that the user needs to fill that in. For example:

```suggestion
WORK_DIR=[INSERT PATH IN HOME DIR]
```
or
```suggestion
WORK_DIR=/home/[USERNAME]/hpc_workshop_october_2024
```


Same with the above PBS options with paths